bHY6yh-d63395/cod-vyg67d_bHY6yh-d63395.xml

```diff
@@ -0,0 +1,160 @@
+            <p xml:id="bHY6yh-d63395-d1e114">
+               De uanilo<lb ed="#O" break="no"/>quio 
+               et multiloquio Rubrica. 11
+            </p>
+            <p xml:id="bHY6yh-d63395-d1e121">
+               <lb ed="#O"/><ref xml:id="bHY6yh-d63395-Rd1e118">SAlomon in prouerbiis. capitulo 15.</ref> ¶ <quote xml:id="bHY6yh-d63395-Qd1e127">Re<lb ed="#O" break="no"/>sponsio 
+               molis frangit iram.
+               <lb ed="#O"/>sermo durus suscitat furorem. ¶ 
+               <lb ed="#O"/>lingua placabilis lignum uite.</quote> 
+            </p>
+            <p xml:id="bHY6yh-d63395-d1e137"> 
+               ¶ <ref xml:id="bHY6yh-d63395-Rd1e135">16. 
+               <lb ed="#O"/>capitulo</ref> <quote xml:id="bHY6yh-d63395-Qd1e139">Qui sapiens corde est appellabitur 
+               <lb ed="#O"/>prudens et qui dulcis eloquio maio<lb ed="#O" break="no"/>ra 
+               percipiet.</quote>
+            </p>
+            <p xml:id="bHY6yh-d63395-d1e148"> 
+               ¶ <ref xml:id="bHY6yh-d63395-Rd1e148">17. capitulo</ref> <quote xml:id="bHY6yh-d63395-Qd1e150">Cum obsecratio<lb ed="#O" break="no"/>nibus 
+               loquitur pauper et diues 
+               <lb ed="#O"/>effabitur rigide.</quote>
+            </p>
+            <p xml:id="bHY6yh-d63395-d1e158">
+               <!--line bumps down-->
+               <lb ed="#O"/><ref xml:id="bHY6yh-d63395-Rd1e162">Ysocrates in <!--line bumps up--> exhortationlbus. capitulo 2.</ref> <!--line bumps back down--> <quote xml:id="bHY6yh-d63395-Qd1e162">Esto quidem modo su<lb ed="#O" break="no"/>auis 
+               uerbo uero facetus. Est autem 
+               <lb ed="#O"/>suauitas quidem salutare occurren<lb ed="#O" break="no"/>tes. 
+               facetie uero alloqui eos lepi<lb ed="#O" break="no"/>dibus 
+               uerbis.</quote>
+            </p>
+            <p xml:id="bHY6yh-d63395-d1e176">
+               <!--line bumps down-->
+               <lb ed="#O"/><ref xml:id="bHY6yh-d63395-Rd1e182">IEsus filius <!--line bumps up--> sirach in ecclesiast. capitulo 5.</ref> <!--line bumps back down--> ¶ <quote xml:id="bHY6yh-d63395-Qd1e180">Prosequatur te 
+               <lb ed="#O"/>uerbum pacis et iustitie.</quote>
+            </p>
+            <p xml:id="bHY6yh-d63395-d1e186"> 
+               ¶ <ref xml:id="bHY6yh-d63395-Rd1e192">6. capitulo</ref> 
+               <lb ed="#O"/><quote xml:id="bHY6yh-d63395-Qd1e188">Uerbum dulce multiplicat amicos 
+               <lb ed="#O"/>et mitigat inimicos. ¶ lingua 
+               <lb ed="#O"/>gratiosa in bono homine abundat.</quote>
+            </p>
+            <p xml:id="bHY6yh-d63395-d1e198">
+               <lb ed="#O"/><ref xml:id="bHY6yh-d63395-Rd1e208">Ouidius de arte amandi. libro 
+               <lb ed="#O"/>2.</ref> ¶ <quote xml:id="bHY6yh-d63395-Qd1e202">Asperitas odium seuaque bella 
+               <pb ed="#O" n="136-r"/>
+               <lb ed="#O"/>monet.</quote>
```

Comment:
```suggestion
               <pb ed="#O" n="136-r"/>
               <cb ed="#O" n="a"/>
               <lb ed="#O"/>monet.</quote>
```

Reply:
Is there a reason why this is a redo of #893

Reply:
I thought I had accidentally put rubric 11 on the same branch as other rubrics but apparently that isn't the case so this can be dismissed.